### PR TITLE
fix(openchallenges): remove Sage from sponsor list

### LINKED ISSUE #2285: [Task] Update `OpenChallenges is made possible by...`
### What product(s) is this story for?

OpenChallenges

### As a user, I want

NA

### Description

Luca suggested to make the distinction that Sage is the author/builder of OC and that NIH made OC possible. Hence, the goal of this ticket is to remove Sage Bionetworks logo from this section of the OC home page.

### Acceptance criteria

_No response_

### Tasks

_No response_

### Anything else?

_No response_

### Have you linked this story to a GitHub Project?

- [X] I have linked this story to a GitHub Project and set its metadata.

### PR DESCRIPTION
**Pull request by vpchung**
Fixes #2285 

## Preview

Screenshot 2023-10-30 at 11 44 53 AM

### CODE REVIEW

**Review by tschaffter — COMMENTED** [review]
What about setting the style of "OpenChallenge is made possible by" as a title similar to the other sections to increase the importance of this section?

**Comment by tschaffter** [comment]
@vpchung Let's make the ITCR clickable to open a new tab with https://itcr.cancer.gov/

**Comment by vpchung** [comment]
> Let's make the ITCR clickable

I like that! Done.

File: libs/openchallenges/home/src/lib/sponsor-list/sponsor-list.component.html [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-          title="Sage Bionetworks"
-        />
+        <a
+          href="https://itcr.cancer.gov/"
+          title="NCI Informatics Technology for Cancer Research"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          <img
+            [src]="(itcr$ | async)?.url"
```
Comment (tschaffter) [inline libs/openchallenges/home/src/lib/sponsor-list/sponsor-list.component.html:13]: A more descriptive variable name would be `itcrImage$`.

```suggestion
            [src]="(itcrImage$ | async)?.url"
```

**Review by tschaffter — APPROVED** [review]
I made one suggestion, otherwise it's good to go! Thanks!

**Pull request merged**